### DISCUSSION TITLE
Collecting the feedback from the live session

### DISCUSSION THREAD

**Discussion by felipemontoya** [eduNEXT/tvm#27] [General]
@Alec4r I just wanted to have a place to discuss the changes we will include from the great feedback given in the live session.

We can make a task of re watching the video to catch more things, but here is what I have from memory:

- improve readme by having the projects explanation at the top
- improve readme by having the project example also use a name
- make the projects command require a directory argument (use current directory as default). Probably this is shared with the project name.
- improve error message when "use V1.2.3" does not find the version because it is not installed.
- Make the default for `tvm list` a little larger. Make the coloring scheme more clear.
- explain in a console log why we need the sudo password when it is asked when installing

Probably more things came up after I left.

**Comment by Alec4r** [comment]
[[Tutor Live Dev] Introducing the Tutor Environment Manager ("tvm") by eduNEXT](https://www.youtube.com/watch?v=Ig3T3QG7g7U)
